kitsune/search/v2/documents.py

```diff
@@ -137,12 +137,37 @@ def get_model(cls):
         return wiki_models.Document
 
 
+class UserInnerDoc(InnerDoc):
+    id = field.Integer()
+    username = field.Keyword()
+
+
+class ProductInnerDoc(InnerDoc):
+    id = field.Integer()
+    title = field.Keyword()
+    slug = field.Keyword()
+
+
+class TopicInnerDoc(InnerDoc):
+    id = field.Integer()
+    title = field.Keyword()
```

Comment:
Our topic titles can be localised too, should these also be indexed as a multi-locale Object?

Reply:
Yes! Why are we treating both title and product as an object and not use them directly in the question doc?

Reply:
Shouldn't title be `Text` instead of `Keyword`?

Reply:
In case we do need an object, you can squash `TopicInnerDoc` and `ProductInnerDoc` together into a single doc like `SumoTitleInnerDoc` (or something with a nicer name :) )